coderd/provisionerjobs_internal_test.go

```diff
@@ -115,3 +130,277 @@ func TestConvertProvisionerJob_Unit(t *testing.T) {
 		})
 	}
 }
+
+func Test_logFollower_completeBeforeFollow(t *testing.T) {
```

Comment:
It'd be good to have a version of this test that runs in postgres too just in case there are differences between the in-memory pubsub implementation and the postgres implementation.

Reply:
Already exists in `provisionerjobs_test.go` I have this one as well because it can assert that we don't query the job, something it's hard to assert with Postgres/FakeDB.